CHANGELOG.rst

```diff
@@ -3,19 +3,31 @@
 0.40.0
 ------
 
-Renku ``0.40.0`` fixes internal KG and Triples Store performance issues.
+Renku ``0.40.0`` introduces UI performance improvements and fixes internal KG and Triples Store performance issues.
+
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**✨ Improvements**
+
+- 🚀 **UI**: Reduce the compiled bundle size to improve performance of the UI (`#2818 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2818>`_,
+  `#2827 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2827>`_, `#2832 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2832>`_)
+- 🚀 **UI**: [Dashboard] Speed up showing the warning for non-indexed projects (`#2824 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2824>`_)
+- 🛠️ **UI**: [Projects] Use the KG API to update a project's metadata for the following cases: visibility, keywords and description (`#2793 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2793>`_)
```

Comment:
Ideally this line would be under "Internal Changes". But if changing it is going to trigger re-running long/broken tests, then it's not worth changing.

Reply:
Right, though I think it fixes inconsistencies in the UI after an update 😄